src/openfermion/utils/_operator_utils.py

```diff
@@ -290,3 +290,61 @@ def save_operator(operator, file_name=None, data_directory=None,
     with open(file_path, 'wb') as f:
         marshal.dump((operator_type, dict(zip(tm.keys(),
                                           map(complex, tm.values())))), f)
+
+
+def reorder(Operator, order_function,num_modes=None, reverse=False):
```

Comment:
Here I would use lowercase to conform with our general style (`operator` instead of `Operator`). We generally use lowercase for variables.